EtherealCereal/EtherWrapper.m

```diff
@@ -46,7 +48,7 @@ - (NSString * _Nonnull)signMessage:(NSData * _Nonnull)message withKey:(NSData *
 
     const unsigned char *private_key = privateKey.bytes;
     int recid = 0;
-    const secp256k1_context *ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
+    secp256k1_context *ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
```

Comment:
do you remember why was it const @Elland  ?

Reply:
Yes, I just wanted to be sure it wasn't changed, but then it is changed when we pass it to the destroy method, so `const` was being ignored by the compiler.